### PR TITLE
Added manufacturer id and maintainer for eachine.

### PR DESCRIPTION
**Pull request by mikeller**
Fixes betaflight/betaflight#10002.

@eachinerc: Am I right to assume that you will be the target maintainer for eachine?

### CODE REVIEW

File: Manufacturers.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ This is the official list of manufacturer ids (`manufacturer_id` in the target c
 |DIAT|Diatone Innovations|https://www.diatoneusa.com/|
 |DRCL|dronercland|https://www.instagram.com/dronercland/|
 |DYST|DongYang Smart Technology Co.,Ltd (dys)|http://www.dys.hk/|
+|EACH|eachine|https://www.eachine.com/|
```
Comment (McGiverGim) [inline Manufacturers.md:26]: Eachine must not be uppercase?

**Comment by EachineRC** [comment]
> Fixes [betaflight/betaflight#10002](https://github.com/betaflight/betaflight/issues/10002).
> 
> @EachineRC: Am I right to assume that you will be the target maintainer for eachine?

Yes, I am everyone’s goal maintainer.

File: Manufacturers.md (hunk 1 above)
Comment (EachineRC) [reply Manufacturers.md:26]: Of course.
 "|EACH|EACHINE|https://www.eachine.com/|"

Comment (mikeller) [reply Manufacturers.md:26]: I got this from the copyright notice on the official website - which is where I normally go for the official company name:

image

Looking at https://www.eachine.com/About-Us-a-1.html, there seems to be a mix of casings used across the page, so not sure what to use:

image

Will go with what we have for now, can always update when this has been cleaned up.

**Pull request merged**